Sources/SafeDICore/Models/TypeDescription.swift

```diff
@@ -476,9 +476,8 @@ extension TypeSyntax {
                 returnType: typeIdentifier.returnClause.type.typeDescription)
 
         } else {
-            assertionFailure("TypeSyntax of unknown type. Defaulting to `description`.")
```

Comment:
We haven't gotten any value out of this `assertionFailure` in multiple years of development on this type across multiple projects. Time to remove.